php_apc.c

```diff
@@ -89,6 +98,21 @@ static void php_apc_init_globals(zend_apcu_globals* apcu_globals)
 
 /* PHP_INI */
 
+static PHP_INI_MH(OnUpdatePercentage)
+{
+	double s = zend_strtod(ZSTR_VAL(new_value), NULL);
+
+	if ((s < 0) || (s > 100)) {
+		php_error_docref(NULL, E_CORE_ERROR, "%s must be a value from 0.0 - 100.0", ZSTR_VAL(entry->name));
+		return FAILURE;
+	}
+
+	double *p = (double *) ZEND_INI_GET_ADDR();
```

Comment:
Why does this use a different approach (ZEND_INI_GET_ADDR instead of using APCG) than the other ini settings?